bc_obps/registration/fixtures/mock/v1/operation.json

```diff
@@ -0,0 +1,472 @@
+      "documents": [],
+      "submission_date": "2024-01-18T15:27:00.000Z",
+      "point_of_contact": 1,
+      "name": "Operation 10",
+      "type": "Single Facility Operation",
+      "naics_code": 21,
+      "opt_in": false,
+      "bcghg_id": "23219990009",
+      "regulated_products": [],
+      "status": "Draft",
+      "created_at": "2024-1-24T15:27:00.000Z",
+      "activities": [1, 5],
+      "registration_purpose": "Reporting Operation"
+    }
+  },
+  {
+    "model": "registration.operation",
+    "pk": "7d3fc7d1-0504-4ee4-a9c5-447f4b324b57",
+    "fields": {
+      "operator": "685d581b-5698-411f-ae00-de1d97334a71",
+      "documents": [],
+      "submission_date": "2024-01-19T15:27:00.000Z",
+      "point_of_contact": 1,
+      "name": "Operation 11",
+      "type": "Single Facility Operation",
+      "naics_code": 21,
+      "opt_in": false,
+      "bcghg_id": "23219990010",
+      "regulated_products": [],
+      "status": "Draft",
+      "created_at": "2024-1-23T15:27:00.000Z",
+      "activities": [1, 5],
+      "registration_purpose": "OBPS Regulated Operation"
+    }
+  },
+  {
+    "model": "registration.operation",
+    "pk": "acf5811e-d521-43f7-a5c7-a6d6dd47bb31",
+    "fields": {
+      "operator": "685d581b-5698-411f-ae00-de1d97334a71",
+      "documents": [],
+      "submission_date": "2024-01-20T15:27:00.000Z",
+      "point_of_contact": 1,
+      "name": "Operation 12",
+      "type": "Single Facility Operation",
+      "naics_code": 21,
+      "opt_in": false,
+      "bcghg_id": "23219990011",
+      "regulated_products": [],
+      "status": "Draft",
+      "created_at": "2024-1-22T15:27:00.000Z",
+      "activities": [1, 5],
+      "registration_purpose": "Reporting Operation"
+    }
+  },
+  {
+    "model": "registration.operation",
+    "pk": "8563da83-0762-4d29-9b22-da5b52ef0f24",
+    "fields": {
+      "operator": "685d581b-5698-411f-ae00-de1d97334a71",
+      "documents": [],
+      "submission_date": "2024-01-21T15:27:00.000Z",
+      "point_of_contact": 1,
+      "name": "Operation 13",
+      "type": "Single Facility Operation",
+      "naics_code": 21,
+      "opt_in": false,
+      "bcghg_id": "23219990012",
+      "regulated_products": [],
+      "status": "Draft",
+      "created_at": "2024-1-21T15:27:00.000Z",
+      "activities": [1, 5],
+      "registration_purpose": "OBPS Regulated Operation"
+    }
+  },
+  {
+    "model": "registration.operation",
+    "pk": "df62d793-8cfe-4272-a93e-ea9c9139ff82",
+    "fields": {
+      "operator": "4242ea9d-b917-4129-93c2-db00b7451051",
+      "documents": [],
+      "submission_date": "2024-01-22T15:27:00.000Z",
+      "point_of_contact": 2,
+      "name": "Operation 14",
+      "type": "Single Facility Operation",
+      "naics_code": 21,
+      "opt_in": false,
+      "bcghg_id": "23219990013",
+      "created_at": "2024-1-20T15:27:00.000Z",
+      "regulated_products": [1, 2, 3, 4],
+      "status": "Transferred",
```

Comment:
The reg1 e2e tests do appear to be looking for those reg2 statuses, because CI fails when I take them out (and the code snippet below is maybe relevant?). I'll see if I can get @shon-button 's help figuring out where the failing assertions are, but if it's not straightforward, are you okay with me leaving these statuses in for this PR, and I'll add an AC about getting rid of them to  #2627 ?

In /home/briannacerkiewicz/cas-registration/bciers/apps/registration1/e2e/poms/operations.ts:
```
expectedValues = [
              OperationStatus.PENDING,
              OperationStatus.APPROVED,
              OperationStatus.DECLINED,
              OperationStatus.CHANGES_REQUESTED,
              OperationStatus.REGISTERED,
              // Below values are not possible in Registration1
              OperationStatus.CLOSED,
              OperationStatus.TEMPORARILY_SHUTDOWN,
              OperationStatus.TRANSFERRED,
```